.prettierrc

```diff
@@ -0,0 +1,14 @@
+{
+  "experimentalTernaries": true,
+  "printWidth": 80,
+  "tabWidth": 2,
+  "singleQuote": true,
+  "quoteProps": "as-needed",
+  "jsxSingleQuote": true,
```

Comment:
Thoughts on changing this to `false`? I find using double quotes easier as it matches the patterns in Vanilla HTML and other frontend frameworks